cms_status_message/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @property
+    def default_status_msg_title(self):
+        """Some default msg titles."""
+        return {
+            'info': _('Info'),
+            'success': _('Success'),
+            'danger': _('Error'),
+            'warning': _('Warning'),
+        }
+
+    @api.model
+    def add_status_message(self, msg, title='', type_='info',
+                           dismissible=True, session=None):
+        """Inject status message in session.
+
+        @param msg: the message you want to display.
```

Comment:
The standard for docstrings is [to use the sphinx syntax](http://www.sphinx-doc.org/en/1.5.1/domains.html#info-field-lists) AFAIK. Not requirement though.